src/app/services/cinchy-query.service.ts

```diff
@@ -87,7 +88,26 @@ export class CinchyQueryService {
     const params = {
       '@formId': formId ? formId : sessionStorage.getItem('formId')
     };
-    return this.cincyService.executeQuery('Cinchy Forms', 'Get Form MetaData', params);
+    return this.cincyService.executeQuery('Cinchy Forms', 'Get Form MetaDataNew', params);
+  }
+
+  getJsonDataBySectionId(formSectionId): Observable<IGetQuery> {
+    // Get JsonData using sectionId.
+    const params = {
+      '@formSectionId': formSectionId
+    };
+    return this.cincyService.executeQuery('Cinchy Forms', 'Get JsonData', params);
+  }
+
+  // init the subject
+  public stopRequest: Subject<void> = new Subject<void>();
+    
+  getJsonDataByFormId() {
+    // Get form Meta data using form Id
+    const params = {
+      '@formId': sessionStorage.getItem('formId')
+    };
+    return this.cincyService.executeQuery('Cinchy Forms', 'Get JsonDataByFormId', params);
```

Comment:
Rename query to "Get Json Metadata By Form Id"

Reply:
actually is this function still being used? if not, please remove

Reply:
Resolved